<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-17T12:44:42 IST" name="Default suite" finished-at="2023-01-17T12:46:04 IST" duration-ms="81561">
    <groups>
    </groups>
    <test started-at="2023-01-17T12:44:42 IST" name="Default test" finished-at="2023-01-17T12:46:04 IST" duration-ms="81561">
      <class name="tests.Test_Play_Store_and_unit_converter">
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:44:42 IST" name="setup" finished-at="2023-01-17T12:44:51 IST" duration-ms="8973" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="playstore_test()[pri:0, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:44:51 IST" name="playstore_test" finished-at="2023-01-17T12:45:44 IST" duration-ms="52818" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- playstore_test -->
        <test-method signature="test_uc()[pri:1, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:45:44 IST" name="test_uc" finished-at="2023-01-17T12:45:57 IST" duration-ms="12688" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_uc -->
        <test-method signature="value_1_test()[pri:2, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:45:57 IST" name="value_1_test" finished-at="2023-01-17T12:45:57 IST" duration-ms="818" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- value_1_test -->
        <test-method signature="value_2_test()[pri:3, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:45:57 IST" name="value_2_test" finished-at="2023-01-17T12:45:58 IST" duration-ms="109" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.Test_Play_Store_and_unit_converter.value_2_test(Test_Play_Store_and_unit_converter.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- value_2_test -->
        <test-method signature="value_3_test()[pri:4, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:45:59 IST" name="value_3_test" finished-at="2023-01-17T12:45:59 IST" duration-ms="138" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.Test_Play_Store_and_unit_converter.value_3_test(Test_Play_Store_and_unit_converter.java:73)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- value_3_test -->
        <test-method signature="closing_and_uninstalling()[pri:5, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:46:00 IST" name="closing_and_uninstalling" finished-at="2023-01-17T12:46:02 IST" duration-ms="2742" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closing_and_uninstalling -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:tests.Test_Play_Store_and_unit_converter@47caedad]" started-at="2023-01-17T12:46:02 IST" name="teardown" finished-at="2023-01-17T12:46:04 IST" duration-ms="1158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- tests.Test_Play_Store_and_unit_converter -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
